llvm/test/CodeGen/AMDGPU/copy-hoist-no-spills.ll

```diff
@@ -0,0 +1,94 @@
+; NOTE: There must be no spill reload inside the loop starting with LBB0_1:
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx908 -O2 -vectorize-loops -vectorize-slp -amdgpu-early-inline-all=true -amdgpu-function-calls=false < %s | FileCheck %s
```

Comment:
-amdgpu-early-inline-all=true -amdgpu-function-calls=false shouldn't do anything with llc. Don't think -vectorize-loops -vectorize-slp do either, and -O2 is the default

Reply:
Thanks. Fixed.